docs/docs/forms.mdx

```diff
@@ -142,44 +140,16 @@ write stories or rules for this case.
 
 ### Slot Mappings
 
-Rasa Open Source comes with four predefined mappings to fill the slots of a form
-based on the latest user message. Please see
-[Custom Slot Mappings](./forms.mdx#custom-slot-mappings) if you need a custom function
-to extract the required information.
-
-#### from_entity
-
-The `from_entity` mapping fills slots based on extracted entities.
-It will look for an entity called `entity_name` to fill a slot `slot_name`.
-If `intent_name` is `None`, the slot will be filled regardless of intent name.
-Otherwise, the slot will only be filled if the user's intent is `intent_name`.
+:::caution Changed in 3.0
+As of 3.0, [slot mappings](./domain.mdx#slot-mappings) are defined in the `slots` section of the domain.
+This change allows the same slot mapping to be reused across multiple forms, removing any unnecessary duplication.
+Please follow the [migration guide](./migration-guide.mdx#slot-mappings) to update your assistant.
+:::
 
-If `role_name` and/or `group_name` are provided, the role/group
-label of the entity also needs to match the given values. The slot mapping will not
-apply if the intent of the message is `excluded_intent`. Note that you can
-also define lists of intents for the parameters `intent` and `not_intent`.
+#### Unique entity mapping matching
 
-```yaml-rasa
-entities:
-- entity_name
-slots:
-  slot_name:
-    type: any
-    mappings:
-    - type: from_entity
-      entity: entity_name
-      role: role_name
-      group: group name
-      intent: intent_name
-      not_intent: excluded_intent
-forms:
-  your_form:
-    required_slots:
-        - slot_name
-```
-
-In `from_entity` mapping, when an extracted entity uniquely maps onto a slot,
-the slot will be filled even if this slot wasn't requested by the form.
+In `from_entity` mapping, when the attributes of an extracted entity (i.e, entity name, role, group) uniquely map
```

Comment:
```suggestion
In the `from_entity` slot mapping, when the attributes of an extracted entity (i.e, entity name, role, group) uniquely map
```

Reply:
Also: What is "uniquely" - the slot has only this mapping from this entity, or the entity is only referenced by one slot mapping?

Reply:
The slot can have multiple mappings, and some can be shared by other slots. But if the slot contains a mapping that is not found in other slots' mappings, then we consider it unique.  Or the entity can be referenced in multiple mappings, but if the role/group is only specific to a slot, then the mapping is unique to that slot.